src/Hedgehog/Internal/Seed.idr

```diff
@@ -138,6 +138,13 @@ nextDoubleR x y = let g = \l,u => let diff = u - l
                                    in mapFst (\f => l + f * diff) . nextDouble
                    in if x <= y then g x y else g y x
 
+||| Get a fast n-th alternative of a generator.
```

Comment:
I don't understand what this means. Please document in such a way that users don't have to follow links to other libraries or papers.

Reply:
I tried ;-)